bundles/org.openhab.binding.e3dc/.project

```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you remove the project file by intention?

Reply:
Yes, it was accidently added from my parallel E3DC project. I made clean rebase yesterday so it is removed.